src/components/Maps/SearchBar.tsx

```diff
@@ -116,16 +116,17 @@ const SearchBar = ({
             disableEnforceFocus
           >
             <List classes={{ root: styles.sidebarList }}>
-              {!loading && suggestions.map((suggestion) => (
-                <ListItem
-                  key={suggestion.description}
-                  classes={{ root: styles.sidebarOptions }}
-                >
-                  <div {...getSuggestionItemProps(suggestion)}>
-                    <Typography>{suggestion.description}</Typography>
-                  </div>
-                </ListItem>
-              ))}
+              {!loading &&
+                suggestions.map((suggestion) => (
```

Comment:
was this a prettier change? let's not touch files that have nothing to do with your PR if possible

Reply:
+1 to Peter's comment

Reply:
maybe this was changed by `npm run lint`?